source/access/text_mode_access_using_putty.rst

```diff
@@ -28,42 +38,85 @@ VSC clusters.
       <hardware>`, and replace ``login.hpc.kuleuven.be`` accordingly.
 
 
-#. Within the category Session, in the field 'Host Name', type in
-   <vsc-loginnode>, which is the name of the login node of the VSC
-   cluster you want to connect to.
+- Within the category 'Session', in the field 'Host Name', type in
+  <vsc-loginnode>, which is the name of the :ref:`login node of the VSC
+  cluster <hardware>` you want to connect to.
 
    .. figure:: text_mode_access_using_putty/text_mode_access_using_putty_01.png
 
```

Comment:
Can this screenshot (text_mode_access_using_putty_01.png) also be changed to "login.hpc.kuleuven.be", as in the "putty_load_saved_session.png" is the case

Reply:
Sure. I improved two screenshots in 0f475fb and 8d4dfb1